content/getting-started/javascript.textile

```diff
@@ -0,0 +1,131 @@
+* Open up the "dev console":https://ably.com/accounts/any/apps/any/console of your first app before instantiating your client so that you can see what happens.
+
+* Run the following function to instantiate the SDK and establish a connection to Ably. At the minimum you need to provide an authentication mechanism. Use an API key for simplicity, but you should use token authentication in a production app. A @clientId@ ensures the client is identified, which is required to use certain features, such as presence:
+
+```[javascript]
+import * as Ably from 'ably';
+
+async function getStarted() {
+
+  const realtimeClient = new Ably.Realtime({ key: '{{API_KEY}}', clientId: 'my-first-client' });
+
+  await realtimeClient.connection.once('connected');
+    console.log(`Made my first connection!`);
+
+};
+
+getStarted();
+```
+
+You can monitor the lifecycle of clients' connections, but for now just log a message to the console to know that the connection attempt was successful. You'll see the message printed to your console, and you can also inspect the connection event in the dev console of your app.
+
+h2(#step-2). Step 2: Subscribe to a channel and publish a message
+
+Messages contain the data that a client is communicating, such as a short 'hello' from a colleague, or a financial update being broadcast to subscribers from a server. Ably uses channels to separate messages into different topics, so that clients only ever receive messages on the channels they are subscribed to.
+
+* Add the following lines to your @getStarted()@ function to create a channel instance and register a listener to subscribe to the channel. Then run it:
+
+```[javascript]
+const channel = realtimeClient.channels.get('my-first-channel');
+
+await channel.subscribe((message) => {
+  console.log(`Received message: ${message.data}`);
+});
+```
+
+* Use the Ably CLI to publish a message to your first channel: @ably channels publish my-first-channel 'Hello!'@. The message will be received by the client you've subscribed to the channel, and be logged to the console.
+
+* In a new terminal tab, subscribe to the same channel using the CLI: @ably channels subscribe my-first-channel@. Publish another message using the CLI and you will see that it's received instantly by the client you have running locally, as well as the subscribed terminal instance.
+
+h2(#step-3). Step 3: Join the presence set
+
+Presence enables clients to be aware of one another if they are present on the same channel. You can then show clients who else is online, provide a custom status update for each, and notify the channel when someone goes offline.
+
+* Add the following lines to your @getStarted()@ function to subscribe to, and join, the presence set of the channel. Then run it:
+
+```[javascript]
+await channel.presence.subscribe((member) => {
+  console.log(`Event type: ${member.action} from ${member.clientId} with the data ${JSON.stringify(member.data)}`)
+});
+
+await channel.presence.enter("I'm here!");
```

Comment:
I think using a string here but JSON in the CLI is confusing, we should just use a string in each case (I think data that says `status = online` is redundant, because if you're offline you won't be in the presence set, so there is not really a case where you have entered `status = offline` in the member data)

Reply:
I agree, however it seems the CLI currently only takes a JSON object for data. I've updated the text away from "online" whilst we fix the CLI.